apps/vast/config.yaml.example

```diff
@@ -0,0 +1,9 @@
+vast: "localhost:42000"
+vast_binary: "vast"
+threatbus: "localhost:13370"
+snapshot: 30
+loglevel: DEBUG
+retro_match: true
+unflatten: true
+transform_context: fever alertify --alert-prefix 'MY PREFIX' --extra-key my-ioc --ioc %ioc # optional. remove the field if you don't want to transform sighting context
+sink: STDOUT # optional. remove the field if you simply want to report back sightings to Threat Bus
```

Comment:
I would not go with inline comments for the already overly long command line. Suggestion: just move them above the respective config parameter.